### PR TITLE
Tweak tito config

### PR DESCRIPTION
**Pull request by vrutkovs**
This will make a package release as simple as:
```
tito tag
git push origin && git push origin atomic-reactor-<version>-<release>
tito release rhel7
```

Note, that 
 1. tito required those ugly tags to be created
 1. relies on version-release tarballs instead of a particular commit

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
-rel-eng/releasers.conf
```
Comment (lcarva) [inline .gitignore]: What was this for anyways?

File: rel-eng/packages/atomic-reactor [hunk 2]
```diff
@@ -0,0 +1 @@
+1.6.19-1 ./
```
Comment (lcarva) [inline rel-eng/packages/atomic-reactor]: Is this supposed to be latest version-release? If so, it's outdated.

File: rel-eng/releasers.conf [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[rhel7]
+releaser = tito.release.DistGitReleaser
+branches = eng-rhel-7
```
Comment (lcarva) [inline rel-eng/releasers.conf]: Is this internal information?

File: atomic-reactor.spec [hunk 4]
```diff
@@ -38,7 +38,8 @@ Summary:        Improved builder for Docker images
 Group:          Development/Tools
 License:        BSD
 URL:            https://github.com/%{owner}/%{project}
-Source0:        https://github.com/%{owner}/%{project}/archive/%{commit}/%{project}-%{commit}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+BuildRoot:      %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
```
Comment (lcarva) [inline atomic-reactor.spec]: koji-containerbuild uses this:

```
%{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
```

The `-XXXXXX` is odd, but honestly I have no idea which one is better.

File: rel-eng/releasers.conf (hunk 3 above)
Comment (vrutkovs) [inline rel-eng/releasers.conf]: Yes, but I think its safe to have this set (dockpulp has it too)

File: rel-eng/packages/atomic-reactor (hunk 2 above)
Comment (vrutkovs) [inline rel-eng/packages/atomic-reactor]: Updated

File: .gitignore (hunk 1 above)
Comment (vrutkovs) [inline .gitignore]: Previously `releasers.conf` was ignored, as it contained information about dist-git and internal COPR. Tito needs this file, so the releasing engineer had to have this file there.

I removed the copr configuration and now `releasers.conf` contains only a distgit releaser and the only internal info there is branch name

File: atomic-reactor.spec (hunk 4 above)
Comment (vrutkovs) [inline atomic-reactor.spec]: The `-XXXXX` version is recommended on a wiki page and it won't require removing it in `%clean`. Not sure which is better to be honest

Comment (twaugh) [inline atomic-reactor.spec]: Don't use BuildRoot.
https://fedoraproject.org/wiki/Packaging:Guidelines#Tags_and_Sections

Comment (vrutkovs) [inline atomic-reactor.spec]: Removed `BuildRoot`

**Pull request merged**